lib/teiserver/bridge/discord_bridge_bot.ex

```diff
@@ -341,6 +342,62 @@ defmodule Teiserver.Bridge.DiscordBridgeBot do
     end
   end
 
+  @spec update_report(Moderation.Report.t()) :: any
+  def update_report(%{discord_message_id: nil}), do: :ok
+  def update_report(report) do
+      channel =
+      cond do
+        report.type == "actions" ->
+          Config.get_site_config_cache("teiserver.Discord channel #overwatch-reports")
+
+        true ->
+          Config.get_site_config_cache("teiserver.Discord channel #moderation-reports")
+      end
+
+      if channel do
+        case Api.Message.get(channel, report.discord_message_id) do
```

Comment:
Please move this to `Teiserver.Communications` delegated to `Teiserver.Communication.DiscordChannelLib`